underlay/src/main/resources/config/datamapping/sd/entity/person/entity.json

```diff
@@ -3,11 +3,12 @@
   "allInstancesSqlFile": "all.sql",
   "attributes": [
     { "name": "id", "dataType": "INT64", "valueFieldName": "person_id" },
+    { "name": "person_id", "dataType": "INT64"},
```

Comment:
This is already covered by the `id` attribute, why add a second one?

Reply:
This would be explicit for users - since id-in different domains refer to different elements. so yes I needed this.